tasks/copy_to_glacier/README.md

```diff
@@ -393,43 +397,4 @@ DATA
     List = typing.List
     MB = 1048576
     Union = typing.Union
-```
```

Comment:
any reason why this pydoc for sqs_library is removed?

Reply:
Whoops. Fixed.